docs/Releases.md

```diff
@@ -11,7 +11,7 @@ In order to consolidate the information about the current versions, we have intr
     - `Z` is the bugfix version. Upgraded in case a hotfix is needed in some `X.Y.*` version.
 
 ## List of Releases
-
+- [PowerAuth 2022.04](./PowerAuth-2022.04.md)
```

Comment:
Formatting: I would keep an empty line after heading.

Reply:
I will not fix this as part of pull request, this pull request is from `develop` into `master`. Feel free to raise a separate issue for this for the next release.